pepys_admin/maintenance/gui.py

```diff
@@ -0,0 +1,876 @@
+import time
```

Comment:
@robintw - it would be great it we could reduce the size of this file, if there's a tidy way of doing it.

I guess we could start off by pulling out the help content.

Reply:
@IanMayo I've shrunk it somewhat by removing the help text to `help.py`, and putting the `create_column_data` function into `utils.py` (actually, I'll be extending that function significantly in the work I'm going to do tomorrow/Friday, so it may move into a completely separate file after that). That has reduced it by a few hundred lines at least. I think pretty-much everything else makes sense to keep within the main class.

Reply:
Great, thanks.